dockerfiles/packaging/debian/james.service

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=JamesMailServer
+Documentation=http://james.apache.org
+Wants=network-online.target cassandra.service elasticsearch.service
+After=network-online.target cassandra.service elasticsearch.service
+
+[Service]
+Environment=WORKING_DIRECTORY=/var/lib/james
+
+User=root
+Group=root
+
+ExecStart=/usr/bin/java -Dworking.directory=${WORKING_DIRECTORY} -Xmx1024m -Dlogback.configurationFile=/etc/james/logback.xml -jar /usr/share/james/james-server.jar 
+
+StandardOutput=journal
+StandardError=inherit
+
+# Specifies the maximum file descriptor number that can be opened by this process
+LimitNOFILE=65535
```

Comment:
I don't know.
You can remove it IMO, one can override this value with systemd specific files